content/blog/tidymodels-survival-analysis/index.Rmd

```diff
@@ -0,0 +1,129 @@
+
+Now we've added the metrics: yardstick v1.3.0 includes new metrics for assessing censored regression models. Somewhat similar to how metrics for classification models can take class predictions or probability predictions as input, these survival metrics can take predicted survival times or predictions of survival probabilities as input. 
+
+The new metrics are
+
+- Concordance index on the survival time via `concordance_survival()`
+- Brier score on the survival probability and its integrated version via `brier_survival()` and `brier_survival_integrated()`
+- Area under the ROC curve on the survival probabilities via `auc_roc_survival()`
+
+The probability of survival is always defined _at a certain point in time_. We call that time point the evaluation time because it is then also the time point at which we want to evaluate model performance. Metrics that work on the survival probabilities are also called _dynamic metrics_ and you read more about them here:
+
+- [Dynamic Performance Metrics for Event Time Data](https://www.tidymodels.org/learn/statistics/survival-metrics/)
+- [Accounting for Censoring in Performance Metrics for Event Time Data](https://www.tidymodels.org/learn/statistics/survival-metrics-details/)
+
+The evaluation time is also the best example to illustrate the changes necessary to the framework. Most of them were under the hood but the evaluation time is user-facing. Let's take a look at that.
+
+While the need for evaluation times is dependent on type of metric, it is not actually specified as an argument to the metric functions. Like yardstick's other metrics, those take pre-made predictions as the input. So where do you specify it then?
+
+You need to specify it to directly predict survival probabilities, via `predict()` or `augment()`. We introduced the corresponding `eval_time` argument first for fitted models in [parsnip and censored](https://www.tidyverse.org/blog/2023/04/censored-0-2-0/#introducing-eval_time) and have added it now for workflows. 
+
+You also need to specify it for the tuninging functions `tune_*()` from tune and finetune as they will predict survival probabilities as part of the tuning process. 
+
+And lastly, the `eval_time` argument now shows up when working with tuning/resampling results such as in `show_best()` or `autoplot()`. Those changes span the packages generating and working with resampling results: tune, finetune, and workflowsets.
+
+As we said, plenty of changes under the hood but you shouldn't need to notice them. Everything else should work "as usual", allowing the same ease and flexibility in combining tidymodels functionality for censored regression as for classification and regression.
```

Comment:
```suggestion
As we said, plenty of changes under the hood but you shouldn't need to notice them. Everything else should work "as usual," allowing the same ease and flexibility in combining tidymodels functionality for censored regression as for classification and regression.
```

This would be MLA style, but us Americans need not impose if current spelling feels cozier for you.🤪

Reply:
This style makes no sense to me but oh well 🏄